google-codelab-index/google_codelab_index.soy

```diff
@@ -23,23 +24,27 @@
   <div class="card-footer {$category}-footer">
     <span class="card-start {$category}-start">Start</span>
     <span class="card-updated">
-      {if $updated}Updated {$updated}{/if}</span>
+      {if $author}<div>{$author}</div>{/if}
+      {if $updated}<div>Updated {$updated}</div>{/if}
+    </span>
   </div>
 {/template}
 
 
 {template .sortby}
+  {@param sort: string}
   {@param categories: list<string>}
   <div id="sort-by-tabs" class="sort-by-inner">
-    <a href="#" sort="alpha" selected>A-Z</a>
-    <a href="#" sort="recent">Recent</a>
-    <a href="#" sort="duration">Duration</a>
+    <a href="#" sort="alpha" {if $sort == 'alpha'}selected{/if}>A-Z</a>
+    <a href="#" sort="recent" {if $sort == 'recent'}selected{/if}>Recent</a>
+    <a href="#" sort="duration" {if $sort == 'duration'}selected{/if}>Duration</a>
   </div>
   <div class="sort-by-inner">
     <select id="codelab-categories">
-    {foreach $category in $categories}
-      <option value="{$category}">{$category}</option>
-    {/foreach}
+      <option value="">Category</option>
+      {foreach $category in $categories}
+        <option value="{$category}">{$category}</option>
```

Comment:
add `selected` for the selected cat from URL? if any...

Reply:
Thats done in JS later, main reason is that soy doesn't have case insensitive compare :(